action.Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM golang:1.13.4-alpine3.10 as builder
 
 WORKDIR /go/src/github.com/rajatjindal/krew-release-bot
 COPY . .
-RUN CGO_ENABLED=0 GOOS=linux go build --ldflags "-s -w" -o krew-release-bot cmd/action/main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -mod vendor --ldflags "-s -w" -o krew-release-bot main.go
```

Comment:
Is there a particular reason why you use vendor mode?

Reply:
to avoid downloading from internet all the time. (beside the fact that i am based in India, and sometimes my internet connection gets patchy and takes forever to download from internet).